specs/account_spec.rb

```diff
@@ -136,36 +130,51 @@
   end
 end
 
-# TODO: change 'xdescribe' to 'describe' to run these tests
-xdescribe "Wave 2" do
+describe "Wave 2" do
   describe "Account.all" do
     it "Returns an array of all accounts" do
-      # TODO: Your test code here!
-      # Useful checks might include:
-      #   - Account.all returns an array
-      #   - Everything in the array is an Account
-      #   - The number of accounts is correct
-      #   - The ID and balance of the first and last
-      #       accounts match what's in the CSV file
-      # Feel free to split this into multiple tests if needed
+      Bank::Account.all.must_be_instance_of Array
     end
+
+    it "Returns the right numbers of accounts" do
+      num = 0
+      CSV.read("/Users/laurams/ada/homework/BankAccounts/support/accounts.csv").each do |line|
+        num +=1
+      end
+      Bank::Account.all.length.must_equal num
+    end
+
+    # TODO: Your test code here!
+    # Useful checks might include:
+    #   - Account.all returns an array     listo!!!
+    #   - Everything in the array is an Account
+    #   - The number of accounts is correct     listo!!!!
+    #   - The ID and balance of the first and last
+    #       accounts match what's in the CSV file
+    # Feel free to split this into multiple tests if needed
+
   end
 
   describe "Account.find" do
-    it "Returns an account that exists" do
-      # TODO: Your test code here!
+    #I am having trouble with this test sometimes.
+    #I think it's because the way I defined my @accounts_array.
+    #But the majority of times it passes. Wierd!
+    it "Returns an account that exits" do
+       Bank::Account.find(1212)
     end
 
     it "Can find the first account from the CSV" do
-      # TODO: Your test code here!
+      Bank::Account.all[0].id.must_equal 1212
     end
```

Comment:
Since this test is under `describe "Account.find"`, it ought to be using the `find` method. How might you modify it to test what it says it tests?